### ISSUE TITLE
license

### ISSUE THREAD

**Issue by ianbenlolo** [fyangneil/pavement-crack-detection#34]
Hi,

contrary to https://github.com/fyangneil/pavement-crack-detection/issues/22, the default is a restrictive license if not specified. Are there plans to add a permissive license to the project? 

Thank you.

**Comment by fyangneil** [comment]
> Hi,
> 
> contrary to #22, the default is a restrictive license if not specified. Are there plans to add a permissive license to the project?
> 
> Thank you.

Actually, I did not add any license control to this repo

**Comment by ianbenlolo** [comment]
Yes, and i think it falls into default which is restrictive. See ["the default copyright laws apply, meaning that you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work"](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#choosing-the-right-license). 
Is this intentional? 

Sorry for confusion.

**Comment by fyangneil** [comment]
> Yes, and i think it falls into default which is restrictive. See ["the default copyright laws apply, meaning that you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work"](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#choosing-the-right-license). Is this intentional?
> 
> Sorry for confusion.

I have no idea. Did this cause you any problem in using the repo.

**Comment by ianbenlolo** [comment]
Yes, i am unable to use it for my work without a permissive license. I would greatly appreciate you adding a license so I can know how i can use it. [Here is a guide to doing this ](https://opensource.guide/legal/#which-open-source-license-is-appropriate-for-my-project). 
thank you very much.
